src/shared/seccomp-util.c

```diff
@@ -213,6 +231,84 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 "sigreturn\0"
                 "time\0"
         },
+        [SYSCALL_FILTER_SET_FILE_SYSTEM] = {
+                /* Basic file system access (reading file properties, opening/creating files and directories for
+                 * read+write) */
+                .name = "@file-system",
```

Comment:
I think similar split to writable and read/other operations was also discussed in early version of the grouping patches (#3053 and #3157). But it's very rare that a program only reads or receives from network and never writes or sends anything.

Blocking file creation and some mount operations may make sense. BTW, there's also creat().